documentation/src/main/asciidoc/userguide/chapters/query/hql/QueryLanguage.adoc

```diff
@@ -1000,6 +1000,9 @@ First, functions for working with strings:
 |===
 | HQL Function | Purpose | Syntax | Notes on syntax
 
+| `chr()` | Create a character from ASCII code point | `chr(integer)` | Standard ANSI SQL
+| `char()` | Synonym to `chr()` | `char(integer)` | Standard ANSI SQL
+| `character_length()` | Synonym to `length()` | `character_length(string)` | Standard ANSI SQL
```

Comment:
Rollback please.

Reply:
1. AFAIK, neither `chr()` nor `char()` are in the ANSI spec. And for sure we don't need to add _both of them_ to the list of standard HQL functions. And I'm not sure why we would `char()` in these days of Unicode. Should we really be encouraging the use of ASCII?
2. I don't see why we need to add the ugly (with a yucky `_`) and verbose synonym `character_length()` as an alternative to JPA-standard `length()`.

Reply:
Also, `chr()` and `ascii()` are apparently not supported on Derby, and it doesn't look easy to emulate them.